### PR TITLE
Update src/Checkboxes.php

### PR DESCRIPTION
**Pull request by wa7eedem**
add withoutTypeCasting #7

### CODE REVIEW

File: src/Checkboxes.php [hunk 1]
```diff
@@ -25,6 +25,20 @@ public function options(array $options)
         return $this->withMeta(['options' => $options]);
     }
 
+    public function withoutTypeCasting()
+    {
+        return $this->withMeta(['withoutTypeCasting' => true]);
+    }
+
+    private function shouldNotTypeCasting()
```
Comment (m2de) [inline src/Checkboxes.php]: Would you mind renaming this to `shouldNotTypeCast`?

**Comment by wa7eedem** [comment]
honestly I have no Idea what to write in the readme page ...

how about this : 

## Configuration
you can customize how the values from checkbox fields are saved to your database .
By default it's will saved as integer , you can save it as string by using `withoutTypeCasting()`

```php
use Silvanite\NovaFieldCheckboxes\Checkboxes;

Checkboxes::make('Permissions')->options([
    1 => 'Access Admin UI',
    2 => 'Manage Users',
]->withoutTypeCasting()),
```

==== also its better if I include an eloquent example : 



#### Example using eloquent 
you can pass eloquent collection for the options as

```php
use Silvanite\NovaFieldCheckboxes\Checkboxes;

Checkboxes::make('users')
    ->options(
        collect(App\user::get())
            ->mapWithKeys(function($user) {
                return [$user->id => $user->name];
            })
        )
    ->withoutTypeCasting(),
```


 ==== what you think? :)

**Comment by m2de** [comment]
Anything will be great, I can tweak it when I go to merge and release the changes. Thanks again for your efforts!

**Comment by m2de** [comment]
Thanks very much, have tweaked a little and pushing a release shortly to resolve #7.

**Comment by wa7eedem** [comment]
thanks for the merge :)

**Pull request merged**